src/types.ts

```diff
@@ -1,4 +1,4 @@
-import { ClientBase, QueryArrayResult, QueryResult, QueryArrayConfig, QueryConfig } from 'pg'
+import { ClientBase, ClientConfig, QueryArrayResult, QueryResult, QueryArrayConfig, QueryConfig } from 'pg'
 import { TlsOptions } from 'tls'
```

Comment:
**suggestion:** As you removed the last reference to `TlsOptions` from this file, you should also clean this `import`